src/features/metadata-instance-editor/__tests__/Instance.test.js

```diff
@@ -718,3 +723,204 @@ describe('features/metadata-instance-editor/fields/Instance', () => {
         });
     });
 });
+
+// New RTL tests
+const testFields = [
+    {
+        id: 'field1',
+        type: 'string',
+        key: 'stringfield',
+        displayName: 'String Field',
+        description: 'example of a string field',
+    },
+    {
+        id: 'field2',
+        type: 'float',
+        key: 'floatfield',
+        displayName: 'Float Field',
+        description: 'example of a float field',
+    },
+];
+
+const testData = {
+    stringfield: 'some string',
+    floatfield: 1.0,
+};
+
+const getBaseProps = (props = {}) => ({
+    id: 'test-instance',
+    canEdit: true,
+    data: { ...testData },
+    template: {
+        id: 'template-1',
+        displayName: 'Test Template',
+        fields: [...testFields],
+        templateKey: 'testTemplateKey',
+    },
+    intl: createIntl({ locale: 'en' }),
```

Comment:
shouldnt need to do this either because its passed into the intlprovider from the custom testing library